docs/source/crud/piccolo_crud.rst

```diff
@@ -196,6 +196,24 @@ For example, to return results 11 to 20:
 
 -------------------------------------------------------------------------------
 
+Bulk delete
+-----------
+
+To specify which records you want to delete in bulk, pass a query parameter 
+like this ``__ids=1,2,3``, and you be able to delete all results whose ``id`` 
+is in the query params.
+
+A query which delete movies with ``id`` pass in query parameter:
+
+.. code-block::
+
+    DELETE https://demo1.piccolo-orm.com/api/tables/movie/?__ids=1,2,3
+
+.. warning:: To be able to provide a bulk delete action, we must set 
+ ``allow_bulk_delete`` to ``True``.
+
+-------------------------------------------------------------------------------
+
```

Comment:
It would be good for us to expand upon these docs some more at some point. You should be able to do stuff like:

```
DELETE /movie/?__name=foobar
```

Reply:
You're right. We can write that all three methods work.

`DELETE /movie/?__name=foobar` (delete all movies with name foobar)
`DELETE /movie/?__ids=1,2,3` (delete all movies with id 1,2,3)
`DELETE /movie/?__name=foobar&__ids=1,2` (delete all movies with name foobar, but only if id is 1 or 2)